images/linux/scripts/installers/2004/android.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+################################################################################
+##  File:  android.sh
+##  Desc:  Installs Android SDK
+################################################################################
+
+set -e
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/apt.sh
+
+# Set env variable for SDK Root (https://developer.android.com/studio/command-line/variables)
+ANDROID_ROOT=/usr/local/lib/android
+ANDROID_SDK_ROOT=${ANDROID_ROOT}/sdk
+echo "ANDROID_SDK_ROOT=${ANDROID_SDK_ROOT}" | tee -a /etc/environment
+
+# ANDROID_HOME is deprecated, but older versions of Gradle rely on it
+echo "ANDROID_HOME=${ANDROID_SDK_ROOT}" | tee -a /etc/environment
+
+# Create android sdk directory
+mkdir -p ${ANDROID_SDK_ROOT}
+
+# Download the latest command line tools so that we can accept all of the licenses.
+# See https://developer.android.com/studio/#command-tools
+wget -O android-sdk.zip https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip
+unzip android-sdk.zip -d ${ANDROID_SDK_ROOT}
+rm -f android-sdk.zip
+
+# Add required permissions
+chmod -R a+X ${ANDROID_SDK_ROOT}
+
+# Check sdk manager installation
+/usr/local/lib/android/sdk/tools/bin/sdkmanager --list 1>/dev/null
+if [ $? -eq 0 ]
+then
+    echo "Android SDK manager was installed"
+else
+    echo "Android SDK manager was not installed"
+    exit 1
+fi
+
+# Install the following SDKs and build tools, passing in "y" to accept licenses.
+echo "y" | ${ANDROID_SDK_ROOT}/tools/bin/sdkmanager \
+    "ndk-bundle" \
+    "platform-tools" \
+    "platforms;android-29" \
+    "platforms;android-28" \
+    "platforms;android-27" \
+    "build-tools;29.0.3" \
+    "build-tools;29.0.2" \
+    "build-tools;29.0.0" \
+    "build-tools;28.0.3" \
+    "build-tools;28.0.2" \
+    "build-tools;28.0.1" \
+    "build-tools;28.0.0" \
+    "build-tools;27.0.3" \
+    "build-tools;27.0.2" \
+    "build-tools;27.0.1" \
+    "build-tools;27.0.0" \
+    "extras;android;m2repository" \
+    "extras;google;m2repository" \
+    "extras;google;google_play_services" \
+    "add-ons;addon-google_apis-google-24" \
+    "add-ons;addon-google_apis-google-23" \
+    "add-ons;addon-google_apis-google-22" \
+    "add-ons;addon-google_apis-google-21" \
```

Comment:
`addon-google_apis-google-*` could be removed because we removed platforms 21-24.
(Starting from platform 25, they are included to build-tools, and we don't need to install them separatelly)

Reply:
Done!